plugins/modules/aws_az_info.py

```diff
@@ -90,12 +92,14 @@ def main():
     connection = module.client('ec2', retry_decorator=AWSRetry.jittered_backoff())
 
     # Replace filter key underscores with dashes, for compatibility
-    sanitized_filters = dict((k.replace('_', '-'), v) for k, v in module.params.get('filters').items())
```

Comment:
I *think* because the original version returned a new dict, the priority (`-` vs `_`) was undefined since it would just iterate through the list and the last returned wins

Reply:
Tried this out, definitely last one wins as it is today.